### ISSUE TITLE
[Rosetta Compatible Signature RFC] Expose a signatureToHex in rosetta.js

### ISSUE THREAD

**Issue by robrobbins** [o1-labs/o1js#1404]
This is needed in order to submit a signed <payment, etc..> to the `construction/submit` rosetta endpoint.

My project expects to be able to utilize mina-signer's `client.signTransaction` method to produce a result compatible with the above rosetta endpoint.

I'm going to open a PR now with what I believe to be a solution, so i'll assign this to myself, if i can - if not i'll simply reference this issue in the PR

**Comment by robrobbins** [comment]
closed as I will open a new PR once #1414 is merged to `main`.
